paper-input-container.html

```diff
@@ -277,6 +279,20 @@
         @apply(--paper-input-container-input);
       }
 
+	  .input-content ::content input:focus,
+      .input-content ::content textarea:focus,
+      .input-content ::content iron-autogrow-textarea:focus,
+      .input-content ::content .paper-input-input:focus {
+        @apply(--paper-input-container-input-focus);
+      }
+
+	  .input-content ::content input[invalid],
```

Comment:
This seems like it would be a good idea, but if the input becomes "invalid" due to being required while being empty, `_computeInputContentClass` will skip the addition of `is-invalid` because `_inputHasContent` would be false. https://github.com/PolymerElements/paper-input/blob/master/paper-input-container.html#L581 What are you're thoughts on this?